libsolidity/formal/SMTEncoder.cpp

```diff
@@ -621,8 +621,11 @@ void SMTEncoder::visitFunctionIdentifier(Identifier const& _identifier)
 	auto const& fType = dynamic_cast<FunctionType const&>(*_identifier.annotation().type);
 	if (fType.returnParameterTypes().size() == 1)
 	{
-		defineGlobalVariable(fType.richIdentifier(), _identifier);
-		m_context.createExpression(_identifier, m_context.globalSymbol(fType.richIdentifier()));
+		auto fName = fType.richIdentifier();
```

Comment:
Have you thought about using `identifier` instead of `richIdentifier` (it is an escaped version)?

Reply:
I haven't. Does it include the full signature?

Reply:
Yes that works. Updated.